src/Config.js

```diff
@@ -100,6 +106,17 @@ module.exports = class extends ChainedMap {
               .forEach(name => this.plugin(name).merge(value[name]));
           }
 
+          case 'plugins': {
+            if (Array.isArray(value)) {
+              return value
+                .forEach((plugin, index) => {
+                  if (plugin.name) index = plugin.name;
+                  if (isClass(plugin)) this.plugin(index).plugin(plugin);
+                  else this.plugin(index).init((args) => plugin);
```

Comment:
it would be nice to decide on a function here to tap args again if the plugin provides some sort of standard interface for it.